Ported/combat-bees/Assets/Scripts/Components/BeeProperties.cs

```diff
@@ -13,5 +13,10 @@ public struct BeeProperties : IComponentData
         public float3 TargetPosition;
 
         public float Aggressivity;
+
+        // Exact world space spawn location (guaranteed in nest)
+        public float3 Origin;
```

Comment:
@LauWerUnity : I don't see you using that anywhere. Can we remove this?

Reply:
Yes, forgot about that, applying corrections!